compiler+runtime/include/cpp/jank/runtime/obj/transient_hash_map.hpp

```diff
@@ -2,6 +2,7 @@
 
 #include <jank/runtime/object.hpp>
 #include <jank/runtime/detail/type.hpp>
+#include <jank/runtime/detail/native_persistent_array_map.hpp>
```

Comment:
Please forward declare this here and include it in the cpp file instead.

Reply:
What does the forward declaration look like and what does it replace in `transient_hash_map(runtime::detail::native_persistent_array_map const &m);`?

Reply:
Ohh, that makes sense. Didn't know you could even do that!